src/modules/core/components/Comment/Dialogs/BanComment/BanComment.tsx

```diff
@@ -131,9 +131,11 @@ const BanComment = ({
                 />
               </div>
             </div>
-            <p className={styles.note}>
-              <FormattedMessage {...MSG.note} values={{ unban }} />
-            </p>
+            {!unban && (
+              <p className={styles.note}>
+                <FormattedMessage {...MSG.note} values={{ unban }} />
```

Comment:
Now that we don't have a note for the `unban` version of this dialog, the values prop shouldn't be needed and therefore the select and `unban` copy inside `MSG.note` is also not needed

Reply:
Good pickup, I have cleaned it up now.